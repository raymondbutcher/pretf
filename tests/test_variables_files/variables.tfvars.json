{
    "image_id": "ami-abc123",
    "availability_zone_names": [
        "us-west-1a",
        "us-west-1c"
    ]
}