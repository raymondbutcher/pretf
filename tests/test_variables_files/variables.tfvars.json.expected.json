[
    {
        "name": "image_id",
        "value": "ami-abc123",
        "source": "variables.tfvars.json"
    },
    {
        "name": "availability_zone_names",
        "value": [
            "us-west-1a",
            "us-west-1c"
        ],
        "source": "variables.tfvars.json"
    }
]